ipaserver/plugins/ca.py

```diff
@@ -161,16 +191,34 @@ def execute(self, *keys, **options):
 class ca_show(LDAPRetrieve):
     __doc__ = _("Display the properties of a CA.")
 
-    def execute(self, *args, **kwargs):
+    takes_options = LDAPRetrieve.takes_options + (
+        Flag(
+            'chain',
+            default=False,
+            doc=_('Include certificate chain in output'),
+        ),
+    )
+
+    def execute(self, *keys, **options):
         ca_enabled_check()
-        return super(ca_show, self).execute(*args, **kwargs)
+        result = super(ca_show, self).execute(*keys, **options)
+        set_certificate_attrs(result['result'], options)
+        return result
 
 
 @register()
 class ca_add(LDAPCreate):
     __doc__ = _("Create a CA.")
     msg_summary = _('Created CA "%(value)s"')
 
+    takes_options = LDAPCreate.takes_options + (
+        Flag(
+            'chain',
+            default=False,
+            doc=_('Include certificate chain in output'),
+        ),
```

Comment:
Could we get rid of this copy-pasta? Maybe like this:
```python
_chain_flag = Flag(
    'chain',
    doc=_('Include certificate chain in output'),
)

...

    takes_options = LDAPCreate.takes_options + (
        _chain_flag,
    )
```

(Also note that all flags implicitly have default value of `False`.)